test/functional/Java22andUp/src/org/openj9/test/jep454/downcall/PrimitiveTypeTests2.java

```diff
@@ -301,13 +301,4 @@ public void test_printfFromDefaultLibWithMemAddr_LinkerOption_2() throws Throwab
 		MemorySegment formatSegmt = arena.allocateFrom("\n%d + %d = %d\n");
 		mh.invoke(functionSymbol, formatSegmt, 15, 27, 42);
 	}
-
-	@Test
-	public void test_validateTrivialOption_2() throws Throwable {
```

Comment:
Why do we want to remove this test? Is it not still valid?

Reply:
See https://github.com/eclipse-openj9/openj9/pull/18930#discussion_r1486961732 for the explanation.